Training/2.0 Libraries.md

```diff
@@ -43,7 +49,8 @@ Create a DataFrame using the Pandas library. Your DataFrame should represent the
 
 
 ```
-
+data = {'Patient': [A,B,C], 'Age': [50,60,70], 'Heart condition': [Arrythmia, Heart Failure, Unstable angina]}
+df = pd.DataFrame(data=data)
 ```
 
```

Comment:
Remember to put '' enclosing the letters / words, otherwise python will look for functions in the code that don't appear:
data = {'Patient': ['A','B','C'], 'Age': [50,60,70], 'Heart condition': ['Arrythmia', 'Heart Failure', 'Unstable angina']}
Otherwise this is correct!